File: Selenium/Activities/Activity19.java
package examples;

import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

import io.github.bonigarcia.wdm.WebDriverManager;

public class Activity19 {

    public static void main(String[] args) {

        WebDriverManager.firefoxdriver().setup();

        WebDriver driver = new FirefoxDriver();

        driver.get("https://training-support.net/webelements/alerts");

        System.out.println("Page title: " + driver.getTitle());

        driver.findElement(By.id("confirmation")).click();

        Alert alert = driver.switchTo().alert();

        System.out.println("Alert text: " + alert.getText());

        alert.accept();
        System.out.println("After OK: " + driver.findElement(By.id("result")).getText());

        driver.findElement(By.id("confirmation")).click();

        alert = driver.switchTo().alert();

        alert.dismiss();
        System.out.println("After Cancel: " + driver.findElement(By.id("result")).getText());

        driver.quit();
    }
}

File: Selenium/Activities/Activity19.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

driver = webdriver.Firefox()

wait = WebDriverWait(driver, 10)

driver.get("https://training-support.net/webelements/alerts")

print("Page title:", driver.title)

driver.find_element(By.ID, "confirmation").click()

alert = wait.until(EC.alert_is_present())

print("Alert text:", alert.text)

alert.accept()
print("After OK:", driver.find_element(By.ID, "result").text)

driver.find_element(By.ID, "confirmation").click()

alert = wait.until(EC.alert_is_present())

alert.dismiss()
print("After Cancel:", driver.find_element(By.ID, "result").text)

driver.quit()